RU: add links to redis university courses/learning paths (#1093)
redis/docs: content/operate/rs/_index.md

- [Redis Stack]({{< relref "/operate/oss_and_stack/stack-with-enterprise" >}})
- [Glossary]({{< relref "/glossary" >}})

## Continue learning with Redis University

> See the [Get started with Redis Software learning path](https://university.redis.io/learningpath/an0mgw5bjpjfbe) for courses.

Reviewer: ```suggestion
See the [Get started with Redis Software learning path](https://university.redis.io/learningpath/an0mgw5bjpjfbe) to learn more.
```
